kotest-assertions/kotest-assertions-jsoup/src/jvmTest/kotlin/io/kotest/assertions/jsoup/ResourceLoader.kt

```diff
@@ -0,0 +1,11 @@
+package io.kotest.assertions.jsoup
+
+import java.io.InputStreamReader
+
+object ResourceLoader {
+   fun getFileAsString(relPath: String): String {
+      val fileStream = javaClass.classLoader.getResourceAsStream(relPath)
+      val fileReader: InputStreamReader? = fileStream.reader()
+      return fileReader?.readText() ?: "// error: file not found or null"
+   }
+}
```

Comment:
@Kerooker does the KotlinTest lib already have a way to do this? I put it here, but I don't think it's the best way to do.
If not, maybe we can add as a new feature from the testing core :man_shrugging: